ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -0,0 +1,378 @@
+
+        let settingsCell = MSFListCellState()
+        settingsCell.title = "Settings"
+        let settingsImageView = UIImageView(image: UIImage(named: "ic_fluent_settings_24_regular"))
+        settingsImageView.tintColor = FluentUIThemeManager.S.Colors.Foreground.neutral4
+        settingsCell.leadingView = settingsImageView
+        settingsCell.onTapAction = defaultMenuAction
+
+        let whatsNewCell = MSFListCellState()
+        whatsNewCell.title = "What's new"
+        let whatsNewImageView = UIImageView(image: UIImage(named: "ic_fluent_lightbulb_24_regular"))
+        whatsNewImageView.tintColor = FluentUIThemeManager.S.Colors.Foreground.neutral4
+        whatsNewCell.leadingView = whatsNewImageView
+        whatsNewCell.onTapAction = defaultMenuAction
+
+        let menuSection = MSFListSectionState()
+        menuSection.cells = [statusCell, statusMessageCell, notificationsCell, settingsCell, whatsNewCell]
+
+        let microsoftAccountCell = MSFListCellState()
+        microsoftAccountCell.layoutType = .twoLines
+        microsoftAccountCell.title = "Contoso"
+        microsoftAccountCell.subtitle = "kat.larrson@contoso.com"
+        microsoftAccountCell.accessoryType = .checkmark
+        let orgAvatar = MSFAvatar(style: .group, size: .large)
+        orgAvatar.state.primaryText = "Kat Larrson"
+        microsoftAccountCell.leadingView = orgAvatar.view
+
+        let msaAccountCell = MSFListCellState()
+        msaAccountCell.layoutType = .twoLines
+        msaAccountCell.title = "Personal"
+        msaAccountCell.subtitle = "kat.larrson@live.com"
+        let msaAvatar = MSFAvatar(style: .group, size: .large)
+        msaAvatar.state.primaryText = "kat.larrson@live.com"
+        msaAccountCell.leadingView = msaAvatar.view
+
+        let addAccountCell = MSFListCellState()
+        addAccountCell.title = "Add Account"
+        let addAccountImageView = UIImageView(image: UIImage(named: "ic_fluent_add_24_regular"))
+        addAccountImageView.tintColor = FluentUIThemeManager.S.Colors.Foreground.neutral4
+        addAccountCell.leadingView = addAccountImageView
+        addAccountCell.onTapAction = defaultMenuAction
+
+        let accountsSection = MSFListSectionState()
+        accountsSection.title = "Accounts and Orgs"
+        accountsSection.cells = [microsoftAccountCell, msaAccountCell, addAccountCell]
+
+        return [menuSection, accountsSection]
+    }
+
+    private var leftNavMenuList = MSFList(sections: [])
+
+    private var leftNavAccountLabelsView: UIView {
+        let nameLabel = UILabel()
+        nameLabel.translatesAutoresizingMaskIntoConstraints = false
+        nameLabel.font = UIFont.preferredFont(forTextStyle: .subheadline)
+        nameLabel.text = "Kat Larrson"
+
+        let titleLabel = UILabel()
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel.font = UIFont.preferredFont(forTextStyle: .caption2)
+        titleLabel.textColor = .darkGray
```

Comment:
Nit: the text is a little hard to distinguish in dark mode, can we perhaps use a lighter color?

Reply:
I agree. The chevron is also an issue. We're going to have a task to leverage the SwiftUI Cell to replace this header.